demos/slider.html

```diff
@@ -169,55 +235,90 @@
           var disabled = demoRoot.querySelector('[name="disabled"]');
           var useCustomColor = demoRoot.querySelector('[name="use-custom-color"]');
           var rtl = demoRoot.querySelector('[name="rtl"]');
-          var value = demoRoot.querySelector('.value');
-          var committedValue = demoRoot.querySelector('.committed-value');
-          var sliderEl = demoRoot.querySelector('.mdc-slider');
-          var slider = new mdc.slider.MDCSlider(sliderEl);
```

Comment:
Currently the demo is set up to have 3 type of slider configured by the same control panel, since the control panel is really long (has `max`, `min`, `rtl`, etc...) 
In this case, I do want to select a `demoRoot` that differentiate contents from hero part, but also since there are multiple sliders within that wrapper, I removed the `continuous` in the class and name the variables to identify which one is which.

Does that answer your question?